src/components/Dashboard.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import Display from "./Display";
+
+class Dashboard extends Component {
+  constructor() {
+    super();
+    this.state = {
+      strikes: 0,
+      balls: 0
+    };
+  }
+
+  strike = () => {
+    if (this.state.strikes === 2) {
+      this.setState({ strikes: 0, balls: 0 });
+    } else {
+      this.setState({ strikes: this.state.strikes + 1 });
+    }
+  };
+
+  ball = () => {
+    if (this.state.balls === 3) {
+      this.setState({ balls: 0, strikes: 0 });
+    } else {
+      this.setState({ balls: this.state.balls + 1 });
```

Comment:
Object destructuring would probably make this slightly prettier (the same goes for the `strike` method:
```javascript
ball = () => {
  const { balls } = this.state;
  if (balls === 3) {
    this.setState({ balls: 0, strikes: 0 });
  } else {
    balls++;
    this.setState({ balls });
  }
};
```
or, if you want to be clever while trading off readability:
```javascript
ball = () => {
  const { balls } = this.state;
  if (balls++ === 3) {
    this.setState({ balls: 0, strikes: 0 });
  } else {
    this.setState({ balls });
  }
};
```